src/app/(routes)/loading-test/page.tsx

```diff
@@ -0,0 +1,122 @@
+'use client';
+
+// loading-test 파일 추후 삭제 예정
+
+// TanStack Query 상태를 이용해 로딩 시 SkeletonCard를 보여줌
+import { SkeletonCardLoader } from '@/shared/components/ui/skeleton/SkeletonCardLoader';
+
+// 단독 UI 테스트용 - 로직 없이 단순 렌더링
+import { SkeletonCard } from '@/shared/components/ui/skeleton/SkeletonCard';
+
+// 와인 더미 데이터 (API 연동 전 테스트용)
+import { Wine, getWines } from '@/shared/libs/api/fake-wine-api';
+
+// TanStack Query 상태를 이용해 로딩 시 Spinner를 보여줌
+import { SpinnerLoader } from '@/shared/components/ui/spinner/SpinnerLoader';
+
+// 단독 UI 테스트용 - 로직 없이 단순 렌더링
+import { Spinner } from '@/shared/components/ui/spinner/Spinner';
+
+
+export default function LoadingTest() {
+  return (
+    <div>
+      <p className='text-xl-bold'>로딩 스피너</p>
+
+      <Spinner size='large' color='primary' />
+      <Spinner size='small' color='primary' />
+
+      {/* SpinnerLoader 사용 예시 */}
+      <SpinnerLoader
+        queryKey={['wines']}
+        queryFn={getWines}
+        size='large'
+        color='primary'
+        spinnerWrapperClassName='flex justify-center items-center h-[20rem]'
+        render={(wines) => (
+          <ul className='grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-6'>
+            {wines.map((wine: any) => (
+              <li
+                key={wine.id}
+                className='border rounded-lg p-4 shadow-sm hover:shadow-md transition'
+              >
+                <h2 className='text-lg font-semibold'>{wine.name}</h2>
+                <p className='text-sm text-gray-500 mb-1'>{wine.description}</p>
+                <p className='text-red-500 font-bold'>
+                  {wine.price.toLocaleString()}원
+                </p>
+                <p className='text-yellow-500 text-sm'>⭐ {wine.rating}</p>
+              </li>
+            ))}
```

Comment:
이건 개인 코드 스타일이긴 한데, 저는 `wine` 객체 데이터에 대하여 구조 분해 할당을 한 상태에서 사용하는 걸 추천드립니다.
```suggestion
            {wines.map((wine: any) => 
            const { id, name, description, price, rating } = wine;
            return (
              <li
                key={id}
                className='border rounded-lg p-4 shadow-sm hover:shadow-md transition'
              >
                <h2 className='text-lg font-semibold'>{name}</h2>
                <p className='text-sm text-gray-500 mb-1'>{description}</p>
                <p className='text-red-500 font-bold'>
                  {wine.price.toLocaleString()}원
                </p>
                <p className='text-yellow-500 text-sm'>⭐ {rating}</p>
              </li>
            ))}
```